ShiftPlan.Blazor.Client/ShiftPlan.Blazor.Client.csproj

```
@@ -8,6 +8,16 @@
     <StaticWebAssetProjectMode>Default</StaticWebAssetProjectMode>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Content Remove="Employee.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Include="Employee.json">
```

```suggestion
    <None Include="TestData/*.json">
```

this way you will copy all json file in directory